paddle_billing/Json/JsonFormatProperties.py

```diff
@@ -0,0 +1,6 @@
+def json_format_properties(properties: list[str]):
```

Comment:
Decorator to avoid formatting every JSON key during encoding (in next major release we could fix the related camel case properties and remove this)